ci/main.sh

```
@@ -37,5 +37,5 @@ cd src/tests
 
 LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${GPG21_INSTALL}/lib"
 export LD_LIBRARY_PATH
-env RNPC_GPG_PATH="${GPG21_INSTALL}/bin/gpg" python2 cli_tests.py all --debug
+env RNPC_GPG_PATH="${GPG21_INSTALL}/bin/gpg" python2 cli_tests.py -w
```

I think we should have `-wvf` here for CI.

---

-v yes, but -f no because I would prefer to get all the errors not only first one

---

I kind of prefer CI failing fast/early, but it's OK either way.

---

I would say ``-f`` is useful during development time. CI should show all regressions (so that developer doesn't have to wait again after fixing first one)